src/qt/forms/debugwindow.ui

```diff
@@ -1342,13 +1342,65 @@
                 </widget>
                </item>
                <item row="12" column="0">
+                <widget class="QLabel" name="peerLastBlockLabel">
+                 <property name="toolTip">
+                  <string>Elapsed time since a novel block passing initial validity checks was received from this peer.</string>
+                 </property>
+                 <property name="text">
+                  <string>Last Block</string>
+                 </property>
+                </widget>
+               </item>
+               <item row="12" column="1">
+                <widget class="QLabel" name="peerLastBlock">
+                 <property name="cursor">
+                  <cursorShape>IBeamCursor</cursorShape>
+                 </property>
+                 <property name="text">
+                  <string>N/A</string>
+                 </property>
+                 <property name="textFormat">
+                  <enum>Qt::PlainText</enum>
+                 </property>
+                 <property name="textInteractionFlags">
+                  <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
+                 </property>
+                </widget>
+               </item>
+               <item row="13" column="0">
+                <widget class="QLabel" name="peerLastTxLabel">
+                 <property name="toolTip">
+                  <string>Elapsed time since a novel transaction accepted into our mempool was received from this peer.</string>
```

Comment:
```suggestion
                  <string>Elapsed time since a novel transaction was received from this peer.</string>
```

Reply:
alternative
```suggestion
                  <string>Elapsed time since a novel transaction, accepted into our mempool, was received from this peer.</string>
```